http/src/main/java/community/redrover/mercuryit/MercuryITHttp.java

```diff
@@ -13,6 +13,9 @@ public class MercuryITHttp extends MercuryITRequest<MercuryITHttp> {
     private HttpRequest.Builder request;
     private HttpRequest.BodyPublisher body;
 
+    private final String contentType = "Content-Type";
+    private final String contentValue = "application/json";
```

Comment:
constants names better to make all caps. outside of that all looks OK

Reply:
I know but it's  like this CONTENT_TYPE in code looks not well! I can change it . No problems.